setup.py

```diff
@@ -103,6 +103,24 @@ def get_ext_suffix():
     return sysconfig.get_config_var("EXT_SUFFIX")
 
 
+def get_python_execute():
+    if WINDOWS:
+        return sys.executable
+    # Try to search more accurate path, because sys.executable may return a wrong one,
```

Comment:
From the issue, sys.executable is only wrong when python is executed from gdb. Does it matter here since it is in setup.py?

Reply:
It's obvious wrong when running release workflows, that is why this error was found.